xline/src/storage/kvwatcher.rs

```diff
@@ -288,6 +292,9 @@ pub(crate) trait KvWatcherOps {
 
     /// Cancel a watch from KV store
     fn cancel(&self, id: WatchId);
+
+    /// Get Prev `KeyValue` of a `KeyValue`
+    fn get_prev_kv(&self, kv: &KeyValue) -> Option<KeyValue>;
```

Comment:
Shall we make a better name? This method's name is a little bit confused with `KvStore::get_prev_key`

Reply:
we don't have a method called `KvStore::get_prev_key`, we only have `KvStore::get_prev_kv`, and this method calls `KvStore::get_prev_key`, they did the same thing